_stbt/match.py

```diff
@@ -316,6 +316,15 @@ def _match_all(image, frame, match_parameters, region):
     template = _load_image(image, cv2.IMREAD_UNCHANGED)
     t = template.image
     mask = None
+
+    if t.dtype == numpy.uint16:
+        warn("Reference image %s has 16 bits per channel. Converting to 8 bits."
+             % (template.friendly_name))
+        t = cv2.convertScaleAbs(t, alpha=1.0 / 256)
```

Comment:
I'm not sure that this is correct.  Should the scale factor be `255. / 65535.` == `1./257`?